### DISCUSSION TITLE
Broken sym link attribute

### DISCUSSION THREAD

**Discussion by anta999** [FarGroup/FarManager#728] [Ideas]
How about adding broken link attribute to file highlighting like in far2 fork ?
Поддержка сломаный ссылок в подстветку файлов ? Как в фопке far2l ?

**Comment by yegor-mialyk** [comment]
It's hard to tell whether a link is broken or not. The fact that the destination file or folder is inaccessible does not mean that the link is broken: you may just not have enough permissions, or the link is pointing to a network resource which is inaccessible under your account etc.

**Reply by anta999** [reply]
I think that's enough just for highlighting. just check it with GetFileAttributeW or whatever

**Reply by anta999** [reply]
![broken](https://github.com/FarGroup/FarManager/assets/48290431/c9021ce0-63d4-42db-a999-a472323ea477)

**Comment by alabuzhev** [comment]
Highlighting inaccessible symlinks is a nice idea.
The only problem is that network paths exist and can be used as symlink targets, so it could suddenly become unimaginably slow.

**Reply by anta999** [reply]
Is it a problem to determine whether a link is a network one? Just ignore them. Or check in the background. But is it really necessary? (network links)
